content/scout/image-analysis.md

```diff
@@ -202,3 +202,34 @@ reference documentation:
 
 - [`docker scout quickview`](../engine/reference/commandline/scout_quickview.md)
 - [`docker scout cves`](../engine/reference/commandline/scout_cves.md)
+
+## Vulnerability severity assessment
+
+Docker Scout assigns a severity rating to vulnerabilities based on
+vulnerability data from [advisory sources](./advisory-db-sources.md).
+Advisories are ranked and prioritized depending on the type of package that's
+affected by a vulnerability. For example, if a vulnerability affects an OS
+package, the severity level assigned by the distribution maintainer is
+prioritized.
+
+If the preferred advisory source has assigned a severity rating to a CVE, but
+not a CVSS score, Docker Scout falls back to displaying a CVSS score from
+another source. The severity rating from the preferred advisory and the CVSS
+score from the fallback advisory are displayed together. This means a
+vulnerability can have a severity rating of `LOW` with a CVSS score of 9.8, if
+the preferred advisory assigns a `LOW` rating but no CVSS score, and a fallback
+advisory assigns a CVSS score of 9.8.
+
+Vulnerabilities that haven't been assigned a CVSS score in any source are
+categorized as **Unspecified** (U).
+
+The following table shows the mapping between CVSS scores and severity ratings
+that's generally applicable, but not necessarily guaranteed given the advisory
```

Comment:
I wonder if we should link out to things like this https://nvd.nist.gov/vuln-metrics/cvss
Note that 'Critical' severity is a CVSS 3.0 thing and didnt exist in 2.0, see link above.

Reply:
I guess my overall message is that "We didnt invent this - we just pass through the information"

Reply:
OK, got it. I attempted to add something along these lines. I feel like the more time I spend on this, the less coherent it gets :D